### ISSUE TITLE
Disable progress bar output during RSpec

### ISSUE THREAD

**Issue by NDuggan** [jfelchner/ruby-progressbar#136]
Firstly, I looked in the Wiki but couldn't find anything about this, so not sure if it's a bug or a feature but here goes…

## Current Behavior
When running RSpec the progress bars are output.

### Steps to Reproduce
1. Add a `ProgressBar` to a model method
2. Add spec for method
3. Run RSpec

### Screenshot(s)
![image](https://user-images.githubusercontent.com/3830636/29666350-d68aa1e8-88cf-11e7-8671-6fcb7da2c233.png)

## Expected Behavior or New Behavior
During testing I don't feel it is necessary to show the progress bars (at least by default).

### Workaround
```
before do
  allow(ProgressBar).to receive(:create).and_return(double.as_null_object)
end
```

**Comment by jfelchner** [comment]
Whether or not the progressbar actually shows during a test run is an application-level concern and not logic that should be built into the library.  For one, there are too many variables, for another, some people might _want_ it to run during testing.

What I would suggest would be a variable you can initialize to a different value depending on the environment, then pass that to `ProgressBar` as the `output` option.  In `test`, you can make that a null object, in any other environment, you can make that `$stdout`.

**Comment by jfelchner** [comment]
@NDuggan [a new feature](https://github.com/jfelchner/ruby-progressbar/wiki/Custom-Output-Classes) will be released in 1.9 (probably releasing it tomorrow) which may also be of future use to you.
